.github/workflows/build.yml

```diff
@@ -0,0 +1,96 @@
+name: Build
+
+on:
+  push:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Unit Tests
+
+    strategy:
+      matrix:
+        node-version:
+          - 12.x
+          - 14.x
+        os:
+          - ubuntu-latest
+          - windows-latest
+          - macOS-latest
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: git checkout
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+    
+      - run: npm ci
+      - run: npm run build --if-present
+      - run: npm test
+
+      - name: Calculate code coverage
+        run: npm run coverage
```

Comment:
There is no code coverage in this repo (so this command will fail).

I would suggest to delete the part of the workflow which handles code coverage.

Reply:
Ohh... okay I will update it

Reply:
I had updated it can you check the code please